rpc/handlers.go

```diff
@@ -215,3 +216,49 @@ func (h *Handler) GetTransactionByHash(hash *felt.Felt) (*Transaction, *jsonrpc.
 	}
 	return adaptTransaction(txn), nil
 }
+
+// https://github.com/starkware-libs/starknet-specs/blob/master/api/starknet_api_openrpc.json#L222
+func (h *Handler) GetTransactionReceiptByHash(hash *felt.Felt) (*TransactionReceipt, *jsonrpc.Error) {
+	txn, rpcErr := h.GetTransactionByHash(hash)
+	if rpcErr != nil {
+		return nil, rpcErr
+	}
+	receipt, blockHash, blockNumber, err := h.bcReader.GetReceipt(hash)
+	if err != nil {
+		return nil, ErrTxnHashNotFound
+	}
```

Comment:
We could probably make this faster by running `GetTransactionByHash` and `h.bcReader.GetReceipt(hash)` concurrently

Reply:
Not really worth to spawn a new goroutine. They are both requests to the local db.